xml/System.Numerics/Vector.xml

```diff
@@ -7178,9 +7178,9 @@ Note that this method returns a <xref:System.Single> instead of an integral type
         <Parameter Name="high" Type="System.Numerics.Vector&lt;System.Int16&gt;" RefType="out" Index="2" FrameworkAlternate="net-6.0;net-7.0;net-8.0" />
       </Parameters>
       <Docs>
-        <param name="source">The source vector whose elements are widened into the outputs.</param>
         <param name="dest1">The first output vector, whose elements will contain the widened elements from lower indices in the source vector.</param>
```

Comment:
Looks like we should delete the old param descriptions.

Reply:
Actually the correct param name is shown depending on which version you select, so this is fine.